modules/govuk_jenkins/templates/jobs/data_sync_complete_integration.yaml.erb

```diff
@@ -12,9 +12,15 @@
     builders:
        - shell: |
            # Clear the Sidekiq queues to stop any residual 'stale' jobs running
+           <%- if scope.lookupvar('::aws_migration') %>
+           for host in $(govuk_node_list -c backend); do
```

Comment:
I think we only need to run this once, so I would add `--single-node` to the `govuk_node_list` command.

Reply:
In Carrenza on the redis hosts it did `for host in ...`, and there were multiple hosts... I assumed we'd need to run it on every host, but now you mention it if it's connecting to the central Redis then it's fine.

(I discovered yesterday that `redis-cli` isn't on backend machines in AWS, which makes running it harder (!), and I can't find anywhere that installs it in Puppet, so we'll have to fix that.)

Reply:
Yup, we only have essentially single Redis endpoint, whereas in Carrenza we (confusingly) have two separate instances.